specs/beacon-chain.md

```diff
@@ -348,25 +348,25 @@ def split(lst, N):
     return [lst[len(lst)*i//N: len(lst)*(i+1)//N] for i in range(N)]
 ```
 
+A helper method for readability:
+
+```python
+def clamp(minval, x, maxval):
+    return minval if x < minval else maxval if x > maxval else x
```

Comment:
Shoud be `return minval if x <= minval else maxval if x >= maxval else x`. Otherwise, we have to ensure that `minval 
 < x < maxval` in somewhere else.

Easy-to-read version:
```python
def clamp(minval: int, x: int, maxval: int) -> int:
    if x <= minval:
        return minval
    elif x >= maxval:
        return maxval
    else:
        return x
```